server/src/test/java/org/spine3/server/storage/RecordStorageShould.java

```diff
@@ -194,6 +195,39 @@ public void write_record_bulk() {
         close(storage);
     }
 
+    @Test
+    public void rewrite_records_in_bulk() {
+        final RecordStorage<I> storage = getStorage();
+        final I id1 = newId();
```

Comment:
Could we use a proper `for` statement instead of manual duplication of entries? If we use `for`, we could adjust the number of records to test easily.